cmd/skopeo/copy.go

```diff
@@ -182,7 +188,8 @@ func (opts *copyOptions) run(args []string, stdout io.Writer) error {
 
 	if len(opts.encryptionKeys.Value()) > 0 {
```

Comment:
Please add a check to reject the command if the user specifies `--encrypt-layer` without `--encryption-key`.

Reply:
done.